mne/annotations.py

```diff
@@ -567,6 +567,50 @@ def crop(self, tmin=None, tmax=None, emit_warning=False, verbose=None):
 
         return self
 
+    @verbose
+    def rename(self, mapping, verbose=None):
+        """Rename annotation description(s). Operates inplace.
+
+        Parameters
+        ----------
+        mapping : dict
+            A dictionary mapping the old description to a new description
+            name e.g. {‘1.0’ : ‘Control’, ‘2.0’ : ‘Stimulus’}.
```

Comment:
```suggestion
            name e.g. {'1.0' : 'Control', '2.0' : 'Stimulus'}.
```

just to avoid copy-paste mistakes.